### PR TITLE
wired up login account to augur logo in market card

### PR DESCRIPTION
**Pull request by bthaile**
https://github.com/AugurProject/augur/issues/3002
This ticket isn't completely done, get account reporting history getter is only stubbed out. There is a little UI work when to fully hook up getter results. 

This PR depends on previous login account reporting reducer changes.

image

### CODE REVIEW

File: packages/augur-ui/src/modules/account/selectors/has-stake-in-market.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { createSelector } from 'reselect';
+import { Getters } from '@augurproject/sdk/build';
```
Comment (andrewdamelio) [inline packages/augur-ui/src/modules/account/selectors/has-stake-in-market.ts]: are we supposed to be referencing the build folder?

**Review by andrewdamelio — APPROVED** [review]
Looks good - 1 comment

File: packages/augur-ui/src/modules/account/selectors/has-stake-in-market.ts (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/account/selectors/has-stake-in-market.ts]: good catch, nope. my mistake

Comment (bthaile) [reply packages/augur-ui/src/modules/account/selectors/has-stake-in-market.ts]: fixed

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🎯

 This link will always point to this PR's latest changes: https://peek.run/p37hpz5

**Pull request closed without merging**